test/format/wts.c

```diff
@@ -196,6 +196,36 @@ configure_timing_stress(char **p, size_t max)
     CONFIG_APPEND(*p, "]");
 }
 
+/*
+ * configure_debug_mode --
+ *     Configure debug settings.
+ */
+static void
+configure_debug_mode(char **p, size_t max)
+{
+    CONFIG_APPEND(*p, ",debug_mode=[");
+
+    if (GV(DEBUG_CHECKPOINT_RETENTION))
```

Comment:
Since this is not a boolean type, could do the check where if it's 0 or non-zero? e.g.
`GV(DEBUG_CHECKPOINT_RETENTION) != 0`

Reply:
yup sounds good, addressed here: [051c0d0](https://github.com/wiredtiger/wiredtiger/pull/8608/commits/051c0d0aa5f2e09205bf6c00b7f65f84128e4803)